kliff/trainer/torch_trainer.py

```diff
@@ -0,0 +1,366 @@
+import os
```

Comment:
Any major reason we want to implement the `torch_trainer.py`? Can it be merged with the lightning trainer to simplify things?

Reply:
We can, but the real reason behind `torch_trainer` is to provide access to descriptor based models, and more flexible trainer for further custom training. Lightning trainer, which most will use, is limited to GNNs as 1) they are clear favorite and most sought after, 2) they can be trained on a scale where lightning makes a difference.